File: OCH_Tao/ABC/B/058.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const O = [...input[0]];
const E = [...input[1]];
let l = [];
for (let i = 0; i < O.length; i++) {
  l.push(O[i]);
  if (O.length > E.length && i === O.length - 1) {
    break;
  } else {
    l.push(E[i]);
  }
}
console.log(l.join(""));

File: OCH_Tao/ABC/B/058.py
O = list(input())
E = list(input())
l = []
for i in range(len(O)):
  l.append(O[i])
  if len(O)-len(E)==1 and i==len(O)-1:
    break
  else:
    l.append(E[i])
print("".join(l))